apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2.java

```diff
@@ -85,9 +85,9 @@ private static Coverage transformPartA(MetricRegistry metricRegistry, Beneficiar
 
     setCoverageStatus(coverage, beneficiary.getPartATerminationCode());
 
-    beneficiary
-        .getMedicareCoverageStartDate()
-        .ifPresent(value -> TransformerUtilsV2.setPeriodStart(coverage.getPeriod(), value));
+    TransformerUtilsV2.setPeriodStart(
+        coverage.getPeriod(), beneficiary.getPartACoverageStartDate());
+    TransformerUtilsV2.setPeriodEnd(coverage.getPeriod(), beneficiary.getPartACoverageEndDate());
```

Comment:
We had checks before to only add these if the values existed; do we want the same here? What happens if nulls get added here? If nothing gets output as before that's fine; but we should double check we dont print out the period with empty values or something in the final output.